src/MSBuild/Resources/Strings.resx

```diff
@@ -877,6 +877,105 @@
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_43_GetPropertySwitch" Visibility="Public">
+    <value>  -getProperty:propertyName,...
+                      Write out the value of the property or properties after
+                      evaluation, without executing the build, or if either
+                      the -targets option or the -getTargetResult option is
+                      used, write out the values after the build.
+    </value>
+    <comment>
+      LOCALIZATION: "-getProperty" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_44_GetItemSwitch" Visibility="Public">
+    <value>  -getItem:itemName,...
+                      Write out the value of the item or items after evaluation
+                      without executing the build, or if either the -targets
+                      option or the -getTargetResult option is used, write out
+                      the values after the build.
+    </value>
+    <comment>
+      LOCALIZATION: "-getItem" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_45_GetTargetResultSwitch" Visibility="Public">
+    <value>  -getTargetResult:targetName,...
+                      Write out the output values of the specified targets.
```

Comment:
I wonder if we should specify that that means executing those targets?

Reply:
don't get your idea. Could you give more information?

Reply:
When you specify -getTargetResult:foo, it will execute Target foo even if the build otherwise would not have executed foo just as if you'd specified -t:foo.

Reply:
Updated